modules/quota_manager/versions.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = ">=0.12.6, <0.14"
```

Comment:
@lawliet89 could we keep >=0.12.6 as in all other modules?

https://github.com/terraform-google-modules/terraform-google-project-factory/pull/505/files

Reply:
We've switched to requiring 0.13.